dotnet/src/Functions/Functions.OpenApi/Extensions/OpenApiFunctionExecutionParameters.cs

```diff
@@ -54,6 +54,14 @@ public class OpenApiFunctionExecutionParameters
     /// </summary>
     public bool EnablePayloadNamespacing { get; set; }
 
+    /// <summary>
+    /// Determines whether payload will be included in the RestApiOperationResponse.
+    /// If true, the payload will be included in the response.
+    /// Otherwise the payload will not be included and RestApiOperationResponse.Payload will be null.
+    /// RestApiOperationResponse.IncludesPayload will be set to true if the payload is included in the response.
+    /// </summary>
+    public bool EnablePayloadInResponse { get; set; }
```

Comment:
Is this property necessary? Why don't we always include the payload in the response? There are no expensive computations or allocations associated with it.

Reply:
I am wondering if this could be renamed to something like "EnableRequestPayloadInResponse". Users may think this might be related to response payload. What do you think?

Reply:
@stephentoub This PR is to address an ask that the request payload be included in the `RestApiOperationResponse` (which represents the response from a OpenAPI REST call). This is to support a specific use case where the application needs to display some debug information to end users. The proposed implementation adds a flag so that is behaviour is opt in so we don't include the request payload always as it may be large. Do you think this optimization is needed?

Reply:
> Do you think this optimization is needed?

Do we already have the object that we'd be referencing from the response object, or do we not already have it and would need to otherwise create it?

Is the concern that someone might be keeping a result object alive for a long period of time not realizing it's rooting a large object?

My gut reaction is to not add such a property, and if it proves important, it could always be added later as opt-out.